.github/workflows/build.yml

```diff
@@ -32,3 +32,15 @@ jobs:
       - name: Test with pytest
         run: |
           pytest
+
+      - name: Check installed module can run python linters
+        run: |
+            python src/python/review/run_tool.py setup.py
```

Comment:
File `src/python/review/run_tool.py` isn't a installed module. To install module, you must first run `pip install .`. This will verify the correctness of `setup.py` file.
And then run the installed module: `review setup.py`.